resources/views/Admin/groups/show.blade.php

```diff
@@ -0,0 +1,62 @@
+@extends('layouts.layout', ['title' => 'Group Management'])
+
+@section('sidebar')
+    @include('layouts.sidebar', ['sidebar'=> Menu::get('sidebar_admin')])
+@endsection
+
+@section('content')
+<!doctype html>
+
+<div class="container mt-4">
+      <div class="card card-body">
+        <div class="container">
+          <div class="row">
+            <div class="col">
```

Comment:
Col not needed

Reply:
Nolan and I looked at this and multiple lines of code would need to be added if this 'col' was removed. Instead 'col' was added to the div below